lib/core/src/Cardano/DB/Sqlite.hs

```diff
@@ -442,6 +449,8 @@ instance DefineSeverity DBLog where
         MsgManualMigrationNeeded{} -> Notice
         MsgManualMigrationNotNeeded{} -> Debug
         MsgUpdatingForeignKeysSetting{} -> Debug
+        MsgFoundDatabase _ _ -> Info
+        MsgUnknownDBFile _ -> Notice
```

Comment:
wonder if unknown db file should not be treated more severely

Reply:
It is not critical as the file is just ignored. It won't even prevent the system from working just fine, hence the `Notice`.